bower.json

```diff
@@ -6,7 +6,7 @@
     "Richard Feldman <richard.t.feldman@gmail.com>"
   ],
   "description": "A Node.js interface to the Elm compiler binaries.",
-  "main": "index.js",
+  "main": "src/index.js",
```

Comment:
I think we can actually drop bower support at this point. I don't know that anyone uses it anymore.